docs/reference/release-notes/6.7.asciidoc

```diff
@@ -65,6 +65,16 @@ SQL::
 
 Also see <<breaking-changes-6.7,Breaking changes in 6.7>>.
 
+[float]
+=== Known issues
+
+Java High Level REST Client::
```

Comment:
I would rather that we not classify this under Java High Level REST client as it might lead to someone to overlook the really problematic issue here. There are two problems: we made a change in Elasticsearch to add the Docker build type. This led to two problems. One problem is that a < 6.7 Java High Level REST client can not read main responses on `/` from a cluster that is upgraded to 6.7.0. This is an unfortunate bug. The other problem is that a rolling upgrade from < 6.7 to 6.7 can not be performed when there are any nodes in 6.7.0 running the official Docker container. This is a really bad bug, and the one that is causing significant problems. This is the bug that we want to highlight as a known issue. I would rather that the known issue focus on this problem.

Reply:
Thanks for the details!  I've updated the description of the problems.